edx_event_bus_kafka/internal/consumer.py

```diff
@@ -133,6 +134,16 @@ def consume_indefinitely(self):
         """
         Consume events from a topic in an infinite loop.
         """
+
+        def reset_offsets(consumer, partitions):
+            # This is a callback method used on consumer assignment to handle offset reset logic.
+            if self.offset is None:
+                return
+            for partition in partitions:
```

Comment:
Aren't we going to want different offsets for each partition?

Reply:
hmm, probably. I think it would be easier then to use timestamps instead of offset values.